content/posts/newsletter/2019-11-08.md

```diff
@@ -0,0 +1,125 @@
+
+## Events
+
+The pace of local meetings and sprints is still high within the XMPP community.
+
+pep tells us the story of the "[Sprint in the cold north](https://bouah.net/2019/10/sprint-in-the-cold-north/)", with sauna and crêpes, working on new groupchat bookmarks specification, file transfer interoperability issues, and a future landing page for new XMPP users.
+
+![Stockholm](https://bouah.net/2019/10/stockholm.jpg)
+
+## Software releases
+          
+### Servers
+
+Erlang Solutions has released MongooseIM 3.5.0 ([go check the changelog](https://github.com/esl/MongooseIM/releases/tag/3.5.0)), and published an article on [GDPR in instant messaging](https://www.erlang-solutions.com/blog/mongooseim-designed-with-privacy-in-mind.html).
+
+This month the Ignite Realtime community has released:
+
+* [Openfire 4.4.3](https://discourse.igniterealtime.org/t/openfire-4-4-3-release/)
+*  [inVerse Openfire plugin 5.0.4.1](https://discourse.igniterealtime.org/t/inverse-openfire-plugin-5-0-4-1-released)
+* [User Status Openfire plugin 1.2.2](https://discourse.igniterealtime.org/t/user-status-openfire-plugin-1-2-2-released)
+* [HTTP File Upload plugin 1.1.1](https://discourse.igniterealtime.org/t/http-file-upload-plugin-1-1-1-released/86563)
+
+Marek Foss has announced [https://blog.process-one.net/ejabberd-19-09-1/ ejabberd 19.09.1].
+
+### Clients and applications
+
+Apps for users have been updated:
+
+* [Pàdé 1.4.0](https://discourse.igniterealtime.org/t/pade-1-4-0-released/86232)
+* [BeagleIM 3.3 and Siskin IM 5.3](https://tigase.net/blog-entry/beagleim-33-and-siskin-im-53-released)
+* [Conversations 2.5.12](https://github.com/siacs/Conversations/releases/tag/2.5.12)
+
+Salut à Toi progress notes for [week 42](https://www.goffi.org/b/SwbEdDJyy5udRTzvVCbaND/progress-note) and [week 44](https://www.goffi.org/b/iY6WhxWD7Qkv5ew6VbYSPG/progress-note) have been published.
+
+### Libraries
+
+QXmpp has been released in versions [1.0.1](https://github.com/qxmpp-project/QXmpp/blob/stable/CHANGELOG.md#qxmpp-101-oct-14-2019) and [1.1.0](https://github.com/qxmpp-project/qxmpp/blob/stable/CHANGELOG.md#qxmpp-110-oct-23-2019).
+
+## Extensions and specifications
+
+### Last Call
+
+Title: XMPP Compliance Suites 2020
+
+Abstract: This document defines XMPP application categories for different use
+cases (Core, Web, IM, and Mobile), and specifies the required XEPs
+that client and server software needs to implement for compliance with
+the use cases.
+
+URL: https://xmpp.org/extensions/xep-0423.html
+
+This Last Call begins today and shall end at the close of business on
+2019-11-06.
+
+### Updated
+
+* Version 0.5.0 of XEP-0405 (Mediated Information eXchange (MIX): Participant Server Requirements) has been released.
+* Version 0.3.0 of XEP-0402 (Bookmarks 2 (This Time it's Serious)) has been released.
+
+## Thanks and CTA, Call To Action (remove this mention before publishing)
+
+This XMPP Newsletter is a community collaborative effort.
```

Comment:
Can a native english speaker have a look at this? I think `a collaborative community effort` sounds better.

Reply:
+1

or perhaps something like:

> This XMPP Newsletter is produced collaboratively by the community.

Reply:
Thx